uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,32 @@
+var Promise = require('es6-promise').Promise;
```

Comment:
the file is called `ssp-leave-absences`, but this is for the my leave page only, so i think the name should be something like `ssp-la-my-leave`?